app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,13 @@
+class QuestionsController < ApplicationController
+  # 1ページの表示数
+  PER_PAGE = 20
+    
+  def index
+    @q = Question.ransack(params[:q])
+    @questions = @q.result.page(params[:page]).per(PER_PAGE)
+  end
+
+  def show
+    @questions = Question.find(params[:id])
```

Comment:
1件のデータですので単数形にしましょう。

```suggestion
    @question = Question.find(params[:id])
```

ところで，「質問一覧ページの実装」と記載されていますが，「詳細ページ」まで実装されるのでしょうか？